helpers/calendar/analytics.py

```diff
@@ -36,6 +36,9 @@ def get_events_number_meetings_room_analytics(self, query, start_date, end_date)
                 number_of_meetings.append(0)
             else:
                 for event in all_events:
+                    if event["start"]["dateTime"] < start_date:
```

Comment:
Hello @RonKbS , great work.  About the 3 similar blocks of code, how about you use a function that can be called.?

Reply:
> Hello @RonKbS , great work. About the 3 similar blocks of code, how about you use a function that can be called.?

This has been worked on @nakatuddesuzan.